testjson/testdata/format/dots-v2.out

```diff
@@ -1,1210 +1,1210 @@
-[?25l
-
-   1ms testjson/internal/badmain 
-
+[?25l[0K
+[0K
+   1ms testjson/internal/badmain [0K
+[0K
  0 tests, 1 failure, 1 error
-[?25h[?25l[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K
-
-   1ms testjson/internal/badmain 
-    🖴  testjson/internal/empty 
-
+[0K[?25h[?25l[5A[0K
+[0K
```

Comment:
I think it's pretty cool that you can see the functional change to the logic by looking at the golden file. The control sequences moved to a different place in the output.